indico/modules/events/abstracts/settings.py

```diff
@@ -37,6 +37,24 @@ class BOACorrespondingAuthorType(RichEnum):
     speakers = 'speakers'
 
 
+class BOALinkFormatting(RichEnum):
+    '''LaTeX book of abstracts link format setting
+
+    value is a 2-tuple of strings:
+    first is the hyperref option to use
+    second sets additional tex commands
+
+    See `indico/legacy/webinterface/tpls/latex/inc/document.tpl`
+    for the usage.
+    '''
+
+    frame = ('', '')
+    colorlinks = ('[colorlinks]', '')
+    ocgcolorlinks = ('[ocgcolorlinks]', '')
+    ocgx2colorlinks = ('', '\\usepackage[ocgcolorlinks]{ocgx2}[2017/03/30]')
```

Comment:
Matter of taste, but I would use a raw string (`r'\usepackage.....'`) instead of escaping the backslash.